### DISCUSSION TITLE
call to cuInit returned error 804: Other

### DISCUSSION THREAD

**Discussion by madlazim** [nikosT/Gisola#23] [General]

**Comment by nikosT** [comment]
@madlazim be more exact also here, please. If both discussions concern the same matter, please keep only one.

**Comment by madlazim** [comment]
@nikosT/Gisola
***@***.***>
 Many thanks for your response.
Sorry, the problem has been solved. Only no space in my computer. I deleted
some files in the result directory, then Gisola was able to run again both
automatically and manually.

Best regards,

Madlazim
